<launch>
    <!-- <node>标签：
        pkg: 节点所在的包名；
        type: 启动的程序的名称；
        name: 节点启动后的名称；
        output: 将节点的输出显示在终端（缺省则将输出定向至日志文件）； output="screen": 将节点的标准输出打印到终端屏幕，默认输出为日志文档。 respawn="true": If the node dies, it will automatically be respawned. default is false。 required="true": 必要节点，当该节点终止时，launch文件中的其他节点也被终止。 ns="namespace": 命名空间，为节点内的相对名称添加命名空间前缀。 args="arguments": 节点需要的输入参数。
            argument是另外一个概念，类似于launch文件内部的局部变量，仅限于launch文件使用，便于launch文件的重构，与ROS节点内部的实现没有关系。
            设置argument使用<arg>标签元素，语法如下：
            <arg name="arg-name" default= "arg-value"/>
            launch文件中需要使用到argument时，可以使用如下方式调用：
            <param name="foo" value="$(arg arg-name)" />
            <node name="node" pkg="package" type="type " args="$(arg arg-name)" />
        -->
    <node pkg='copley' type='Controller_DiffDrive.py' name='Controller_DiffDrive' />
    <node pkg='copley' type='Controller_SwitchDrive.py' name='Controller_SwitchDrive' />
</launch>